docs/Concepts/Bonsai.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Bonsai Tries data storage
+---
+
+# Bonsai Tries data storage format
+
+!!! caution
+
+    Bonsai Tries is an experimental feature.
+
+Bonsai Tries is a new data storage layout policy for Besu, designed to reduce storage requirements and increase
+read performance. It is currently an experimental feature, but will replace the current data structure, the
+"Forest of Tries".
```

Comment:
Probably remove, "It is currently an experimental feature, but will replace the current data structure, the
"Forest of Tries"".